org.eclipse.swt.win32.win32.x86/src/org/eclipse/swt/internal/gdip/Gdip.d

```diff
@@ -475,7 +475,7 @@ public:
 
     static void BitmapData_delete (BitmapData* bitmapdata)
     {
-        delete bitmapdata;
+        destroy(bitmapdata);
```

Comment:
As far as I know, `destroy` will not free memory while `delete` did. Not sure if it matters.

Reply:
The BitmapData_delete is paired with this.

     static BitmapData * BitmapData_new ()
     {
         return new BitmapData;
     }

BitmapData is small struct without finalizer. There seems to be no meaning in the destroy. This destroy simply indicates the intention of destruction.

The other destroy is the same.